release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3096,25 +3098,48 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
-//Update with relevant advisory information
-[id="ocp-4-14-0-ga"]
-=== RHSA-2023:5006 - {product-title} {product-version}.0 image release, bug fix, and security update advisory
+[id="ocp-4-14-2"]
+=== RHSA-2023:6837 - {product-title} 4.14.2 bug fix and security update
 
-Issued: 2023-10-31
+Issued: 2023-11-15
 
-{product-title} release {product-version}.0, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:5006[RHSA-2023:5006] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:5009[RHSA-2023:5009] advisory.
+{product-title} release 4.14.2, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:6837[RHSA-2023:6837] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:6840[RHSA-2023:6840] advisory.
 
 Space precluded documenting all of the container images for this release in the advisory.
 
 You can view the container images in this release by running the following command:
 
 [source,terminal]
 ----
-$ oc adm release info 4.14.0 --pullspecs
+$ oc adm release info 4.14.2 --pullspecs
 ----
-//replace 4.y.z for the correct values for the release. You do not need to update oc to run this command.
 
-[id="ocp-4-14-1-ga"]
+[id="ocp-4-14-2-bug-fixes"]
+==== Bug fixes
+
+* Previously, a change to the default security settings for new Microsoft Azure storage accounts in the `eastus` region prevented the installation of {product-title} clusters that use Azure AD Workload Identity in that region. The issue has been resolved in this release. (link:https://issues.redhat.com/browse/OCPBUGS-22651[*OCPBUGS-22651*])
+
+* Previously, the Docker build deployment would fail since the inline Dockerfile hooks did not preserve the modification time of the file that was copied. With this release, the '-p' flag was added to the `cp` command when copying artifacts between containers to preserve timestamps. (link:https://issues.redhat.com/browse/OCPBUGS-23006[*OCPBUGS-23006*])
```

Comment:
```suggestion
* Previously, the Docker build deployment would fail because the inline Dockerfile hooks did not preserve the modification time of the file that was copied. With this release, the '-p' flag was added to the `cp` command when copying artifacts between containers to preserve timestamps. (link:https://issues.redhat.com/browse/OCPBUGS-23006[*OCPBUGS-23006*])
```